MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+public partial class App : PrismApplication
+{
+    public App(IPlatformInitializer initializer = null) : base(initializer) { }
+
+    protected override void OnInitialized()
+    {
+        InitializeComponent();
+
+    }
+
+    protected override void RegisterTypes()
+    {
+        Container.RegisterTypeForNavigation<MainPage, MainPageViewModel>();
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight xaml %}
+<?xml version="1.0" encoding="utf-8" ?>
+<prism:PrismApplication xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                        xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                        xmlns:prism="clr-namespace:Prism.Unity;assembly=Prism.Unity.Forms"
+                        x:Class="ListViewPrism.App">
+</prism:PrismApplication>
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:prism="clr-namespace:Prism.Mvvm;assembly=Prism.Forms"
+             xmlns:local="clr-namespace:ListViewPrism;assembly=ListViewPrism"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView"
+             x:Class="ListViewPrism.Views.MainPage"
+             Title="MainPage">
+    <StackLayout HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand">
+        <syncfusion:SfListView x:Name="listView" ItemSize="70" ItemSpacing="0,0,5,0"
+                             AutoFitMode="Height"
+                             ItemsSource="{Binding ContactsInfo,}" IsStickyHeader="True" 
+                             AllowSwiping="True" IsStickyGroupHeader="True" GroupHeaderSize="50">
+        </syncfusion:SfListView>
+    </StackLayout>
+</ContentPage>    
+{% endhighlight %}
+{% endtabs %}
+
+## Improving ListView performance
+
+The .NET MAUI ListView (SfListView) has been built from the ground up with an optimized view reuse strategy for the best possible performance on the .NET MAUI platform even when loading large data sets. Following techniques are used to improve performance of the `SfListView`:
+
+ * Bind the `ItemsSource` property to an IList<T> collection instead of an IEnumerable<T> collection because IEnumerable<T> collection do not support random access.
+ * The `SfListView` gets refreshed each and every time a new item added into the underlying collection. Because, when adding items at runtime, the `DataSource` gets refreshed. To avoid this behavior, use [BeginInit()](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataSource.DataSource.html#Syncfusion_Maui_DataSource_DataSource_BeginInit) to stop the `RefreshView()` calling in each time, and use [EndInit()](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataSource.DataSource.html#Syncfusion_Maui_DataSource_DataSource_EndInit) to start the `RefreshView()` calling when adding number of finished items.

```

Comment:
The `SfListView` gets refreshed each and every time when a new item is added into the underlying collection.

Reply:
I have modified the line as per the concerns provided

Reply:
I have modified the line as per the concerns provided